posts/inside-rust/leadership-council-repr-selection@5.md

```diff
@@ -0,0 +1,22 @@
++++
+layout = "post"
+date = 2025-03-25
+title = "Leadership Council March 2025 Representative Selections"
+author = "Eric Huss"
+team = "Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>"
++++
+
+The March 2025 selections for [Leadership Council] representatives have been finalized. The compiler team has chosen [Josh Stone] as their new representative. [Eric Huss] and [James Munns] will continue to represent [Devtools] and [Launching Pad] respectively.
```

Comment:
@cuviper: Is there anything you'd like us to say here about you by way of introduction?

Reply:
AFAICS past announcements haven't really done introductions, so I think it's ok to leave it.